share/spice/get_events/get_events.js

```diff
@@ -0,0 +1,73 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_get_events = function(api_result){
+        if (api_result.error) {
+            return Spice.failed('get_events');
+        }
+	var months = ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'June', 'July', 'Aug', 'Sept', 'Oct', 'Nov', 'Dec'];
+
+        Spice.add({
+            id: "get_events",
+            name: "Events",
+            data: api_result.events,
+            meta: {
+                sourceName: "GetEvents",
+                sourceUrl: 'http://getevents.co',
+		itemType: "Upcoming Events"
+            },
+            normalize: function(item) {
+		function getDate(date) {
```

Comment:
Let's move these functions and any other constants,  like `var months` outside of `ddg_spice_get_events` callback definition.